scripts.js

```diff
@@ -1,6 +1,11 @@
 function add_doggo() {
-    var src = "https://dog.ceo/api/breed/shiba/images";
-    show_image("https://dog.ceo/api/breed/shiba/images", 500,500, "Doggo");
+    fetch("https://dog.ceo/api/breed/shiba/images")
+        .then(function(response) {
```

Comment:
This can be a little simpler with an arrow function...
`.then(response => response.json())`